Libraries/Blob/Blob.js

```diff
@@ -149,7 +120,24 @@ class Blob {
    * `new Blob([blob, ...])` actually copies the data in memory.
    */
   close() {
-    BlobModule.release(this.blobId);
+    const BlobManager = require('BlobManager');
+    BlobManager.release(this.data.blobId);
+    this.data = null;
+  }
+
+  /**
+   * Size of the data contained in the Blob object, in bytes.
+   */
+  get size(): number {
```

Comment:
If we remove the ability to mutate a `Blob`, can this just be a plain number property on each instance?

Reply:
These are read-only properties according to the `Blob` API. Replacing with plain property will make them writable.